spec/index.bs

```diff
@@ -1604,7 +1604,14 @@ dictionary GPUImageBitmapCopyView {
 <script type=idl>
 interface mixin GPUProgrammablePassEncoder {
     void setBindGroup(unsigned long index, GPUBindGroup bindGroup,
-                      optional sequence<GPUBufferSize> dynamicOffsets = []);
+                      optional sequence<unsigned long> dynamicOffsets = []);
+
+    void setBindGroup(unsigned long index, GPUBindGroup bindGroup,
+                      Uint32Array dynamicOffsetsData);
```

Comment:
We talked about this in the spec editors meeting and maybe we don't even need this overload either. It doesn't seem that useful. Also, I think passing a uint32array without the extra args would demote into the `sequence<>` overload anyway (though it would be much slower).